README.md

```diff
@@ -41,6 +41,15 @@ Need some help from the Pieces Copilot? Whether it's commenting, fixing a bug, o
 
 ![](https://storage.googleapis.com/pieces_multimedia/PROMOTIONAL/PIECES_FOR_DEVELOPERS/SUBLIME/ask.gif)
 
+### Ask about the current Project/File
+
+- **Ask About the Current Project**: Use the command `Pieces: Ask about the current project` to get to be able to ask the Copilot on your Project.
+- **Ask About the Current File**: Use the command `Pieces: Ask about the current file` to get file-specific assistance.
```

Comment:
Should you add this line: to get file-specific assistance or the current file that is in focus?